src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -0,0 +1,749 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Internal.Runtime.CompilerServices;
+
+namespace System
+{
+    internal static partial class Number
+    {
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe ref struct BigInteger
+        {
+            private const int MaxBlockCount = 35;
+
+            private static readonly uint[] Pow10UInt32Table = new uint[]
```

Comment:
Nit: These should have `s_` prefix according to the corefx coding conventions.

Reply:
Fixed.